shared/common-adapters/zoomable-image.native.tsx

```diff
@@ -10,24 +10,64 @@ const Kb = {
 
 const ZoomableImage = (p: Props) => {
   const {src, style, onChanged, onLoaded} = p
+  const onZoom = onChanged
```

Comment:
in 72 there is a weird race where if you zoom the content container will get resized weirdly and won't be correctly nested in the scrollview. this only happens sometimes but is easy to repo. Instead we do some more explicit sizing which seems to fix the issue